### DISCUSSION TITLE
Primer addons

### DISCUSSION THREAD

**Discussion by echarles** [primer/react#3297] [Show and tell]
Hi 👋 , we are working on components that are not (yet) available in primer react.

We have for now a `Toolbar` and a `Slider`:

- Storybook on https://primer-addons.datalayer.tech
- Npm.js package`@datalayer/primer-addons` https://npmjs.com/package/@datalayer/primer-addons
- Open source code https://github.com/datalayer/primer-addons

We will continue to make the existing components better and add more based on our needs.
Looking forward your feedbacks here or on the Ξ Datalayer repository.

PS: This was driven by the need for a Slider for the React JSON Form repository https://github.com/datalayer/primer-rjsf.

<img width="699" alt="Screenshot 2023-05-18 at 20 32 06" src="https://github.com/primer/react/assets/226720/214ea933-3df5-4b87-aab8-9ec402cb9abd">

<img width="347" alt="Screenshot 2023-05-18 at 20 31 50" src="https://github.com/primer/react/assets/226720/35d333ca-ac38-4069-8a76-954acecc1c0a">

**Comment by lesliecdubs** [comment]
Thanks for sharing @echarles! 🙌🏻
